docs/agents/sematext-agent/releasenotes.md

```diff
@@ -35,6 +39,60 @@ NULL
 
 -->
 
+## Version 3.2.0
+
+Date: October 19, 2023
+
+### New Features
+
+- CronJobs Monitoring: We are happy to introduce metrics collection for CronJobs. Now, you can gain insights into the metrics of your CronJobs. You can check all supported CronJob metrics in [this wiki](https://sematext.com/docs/integration/kubernetes/#cronjob-overview-metrics).
+- Kube-controller Monitoring: We now support the collection of metrics from kube-controller, providing you with valuable data to monitor Kubernetes controller performance. You can check all supported kube-controller metrics in [this wiki](https://sematext.com/docs/integration/kubernetes/#kube-controller-metrics).
+- Kubernetes Pod Labels: We have added support for collecting metrics related to Kubernetes Pod Labels, allowing for more in-depth monitoring of your pods.
+- MySQL General Logs Support: In this update, we introduce support for parsing MySQL general logs.
+
+### Improvements
+
+- API Server: We now monitor the API server even if the nodes are unreachable.
+- Ship Heartbeat Failures: With this improvement, we ensure that heartbeat failures are properly handled and reported.
```

Comment:
@yago-123

Reply:
Ship heartbeat failures is from Etcd, we added a metric from Etcd that represents the heartbeat failures for that component